client/src/components/ModalVideo.js

```diff
@@ -23,7 +15,7 @@ function ModalVideo({
 
 	let modalContent;
 
-	if (url && description) {
+	if (rhythm.url && rhythm.description) {
```

Comment:
All `rhythm.url` mentions need to reference `rhythm.video` as this is what the API returns